packages/create-schemas/tests/fixtures.ts

```diff
@@ -0,0 +1,44 @@
+import { Worker } from "node:worker_threads";
+import { mkdir, rm } from "node:fs/promises";
+import { join } from "node:path";
+import { fileURLToPath } from "node:url";
+import type { OnTestFailedHandler } from "vitest";
+
+const tempFolder = fileURLToPath(
+    new URL("../node_modules/.tmp", import.meta.url)
+);
+export const dataFolder = fileURLToPath(new URL("./data", import.meta.url));
+
+interface CreateTemporaryFolderOptions {
+    onTestFinished: (fn: OnTestFailedHandler) => void;
+}
+
+export async function createTemporaryFolder({
+    onTestFinished
+}: CreateTemporaryFolderOptions): Promise<string> {
+    const id = Math.random().toString(36).substring(2);
```

Comment:
just curious why we define it this way and not use uuid?

Reply:
No real reason. Honestly, let's just use UUID, since the intend will be more easily understood than `Math.random().toString(36).substring(2)`. 👍